base/ubi/9.4-minimal/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.4-1194
+
+SHELL ["/usr/bin/bash", "-euExo", "pipefail", "-O", "inherit_errexit", "-c"]
+
+# Install a minimal subset of packages that *every* runtime image needs.
+RUN microdnf update -y && \
+    microdnf install -y jq && \
```

Comment:
this also used to explicitly install `curl` and `ca-certificates` - are you dropping them intentionally, or relying on it being already present?

Reply:
if the later, stating the dependencies explicitly makes it more resilient to changes

Reply:
curl and ca-certificates are already present.